frontend/src/components/pod/Details.tsx

```diff
@@ -96,38 +97,36 @@ function PodLogViewer(props: PodLogViewerProps) {
   }
   const debouncedSetState = _.debounce(setLogsDebounced, 500, options);
 
-  React.useEffect(
-    () => {
-      let callback: any = null;
+  React.useEffect(() => {
+    let callback: any = null;
```

Comment:
I see that only one part of this function is effectively changed. Can you keep the format that it had before? (makes the change safer and easier to review)